src/layout/DwindleLayout.cpp

```diff
@@ -338,7 +345,42 @@ void CHyprDwindleLayout::onWindowCreatedTiling(CWindow* pWindow) {
 
     const auto PFORCESPLIT = &g_pConfigManager->getConfigValuePtr("dwindle:force_split")->intValue;
 
-    if (*PFORCESPLIT == 0) {
+    bool       horizontalOverride = false;
+    bool       verticalOverride   = false;
+
+    // let user select position -> top, right, bottom, left
+    if (focusDirection != OneTimeFocus::NOFOCUS) {
+        switch (focusDirection) {
```

Comment:
wtf

Reply:
didn't want to go into the forcesplit when someone manually overrides it, and the switches requires every case without giving warnings.
Any suggestions?

Reply:
this does not require a switch statement and can be compressed to `focusDirection % 2 == 0`-like ifs